tutorials/07-voting/README.md

```diff
@@ -0,0 +1,645 @@
+---
+title: 9. Voting Contract
+---
+
+With the advent of blockchain technology and smart contracts,
+it has become popular to try to create decentralized voting mechanisms that allow large groups of users to vote completely on chain.
+This tutorial will provide a trivial example for how this might be achieved by using a resource-oriented programming model.
+Two contracts will allow users to vote on multiple proposals while their voting power is determined by the balance of certain tokens. 
+These so called governance tokens track the user account balance over time, thus enabling insight into the balance just before the creation of a proposal. 
+An administration contract serves for the creation of proposals and provides the ballots.
+
+---
+We'll take you through these steps to get comfortable with the voting contracts.
+
+1. Deploy the contracts to the local blockchain emulator
+2. Create two user accounts
+3. Create voter accounts for these users
+4. Mint tokens to these accounts
+5. Create proposals for users to vote on
+6. Create `Ballot` resources for both voters
+7. Record and cast votes in the central voting contract
+8. Read the results of the vote
+
+## Setup the environment
+
+In this tutorial, we are introducing the [Flow CLI](https://developers.flow.com/tools/flow-cli/index), which allows us to run our code on a local blockchain emulator. 
+Please follow the link and install it on your computer. You should now be able to call it by simply entering `flow`, which will show you a list with all commands.
+The CLI also contains the blockchain emulator, please take a look at the [Flow CLI subsection](https://developers.flow.com/tools/flow-cli/start-emulator) for a brief overview, more details are covered in the [ReadMe](https://github.com/onflow/flow-emulator/#readme).
+
+Once installed, please download the project in the terminal by executing `git clone git@github.com:onflow/flow-playground-tutorials.git`, and then go to the project folder: `cd tutorials/08-voting`. 
+Once there, please execute this command which will create an initial configuration:
+
+```console
+flow init
+```
+
+Then, edit the configuration and add both the `contracts` and the `deployments` section, this is needed for the deployment of the project:
```

Comment:
We need a description about what flow.json is used for and what the different fields are